app/assets/javascripts/discourse/app/components/d-document.js

```diff
@@ -0,0 +1,57 @@
+import Component from "@ember/component";
+import { bind } from "@ember/runloop";
+import { inject as service } from "@ember/service";
+
+export default Component.extend({
+  _boundFocusChange: null,
+  tagName: "",
+  documentTitle: service(),
+
+  didInsertElement() {
+    this.documentTitle.setTitle(document.title);
```

Comment:
what is the reason for this happening before super ? Feels like it is so rare, it should be documented if on purpose

Reply:
Oops! Not intentional. I've pushed a fix.